### PR TITLE
Rustfmt pass

### PR DESCRIPTION
**Pull request by vitvakatu**
It's just an initial pass, give me your opinions about syntax.

### CODE REVIEW

File: src/factory/load_gltf.rs [hunk 1]
```diff
@@ -82,41 +73,27 @@ impl super::Factory {
     }
 
     /// Loads a `glTF` material.
-    fn load_gltf_material(
-        &mut self,
-        mat: &gltf::Material,
-        buffers: &gltf_importer::Buffers,
-        base: &Path,
-    ) -> Material {
+    fn load_gltf_material(&mut self, mat: &gltf::Material, buffers: &gltf_importer::Buffers, base: &Path) -> Material {
```
Comment (kvark) [inline src/factory/load_gltf.rs]: I think the old formatting is better here

**Comment by vitvakatu** [comment]
A pushed second commit with different option for function args. But it's not good.
It's weird behavior actually, default `Tall` value shouldn't provide these results.

**Comment by vitvakatu** [comment]
If you have info how to fix that, let me know

**Comment by vitvakatu** [comment]
Tomorrow, sorry. Going to sleep now

**Pull request merged**